pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -542,8 +542,10 @@ type Model struct {
 	Type string `xml:"type,attr"`
 }
 
+// TODO <target dev='eth0' managed='no' </target>
 type InterfaceTarget struct {
-	Device string `xml:"dev,attr"`
+	Device  string `xml:"dev,attr"`
```

Comment:
unintended whitespace noise?

Reply:
gofmt made me do it - I've added a line below.